src/Credentials/ServiceAccountCredentials.php

```diff
@@ -219,11 +219,20 @@ public function fetchAuthToken(callable $httpHandler = null)
     }
 
     /**
+     * Return the Cache Key for the credentials.
+     * For the cache key format is:
+     * ClientEmail:Scope | Audience:sub.
```

Comment:
Hmm, the pipe format is hard to read. How about this:
```suggestion
     * For the cache key format is one of the following:
     * ClientEmail.Scope[.Sub]
     * ClientEmail.Audience[.Sub]
```